src/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 find_package(Vulkan REQUIRED)
 
-add_library(vuh SHARED device.cpp error.cpp instance.cpp utils.cpp)
+set(vuh_BUILD_TYPE SHARED CACHE STRING "STATIC or SHARED")
```

Comment:
Could you move this to the root CMake please?  Next to the options.

Reply:
Done